templates/download/raspberry-pi/index.html

```diff
@@ -8,175 +8,146 @@
+      <hr />
+      <p class="u-responsive-realign">
+        <a class="p-button--positive" href="/download/raspberry-pi/thank-you?version={{ releases.latest.full_version }}&amp;architecture=desktop-arm64+raspi" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Desktop', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });" class="p-button u-no-margin--bottom">Download Ubuntu Desktop 23.10</a>
+        &nbsp;
+        <span class="u-text--muted">ISO image, 2.4GB</span>
       </p>
-      <p>
+      <p>Minimum 4GBs of RAM required</p>
+      <hr />
+      <p class="u-responsive-realign">
         <a href="/download/raspberry-pi/thank-you?version={{ releases.latest.full_version }}&amp;architecture=server-arm64+raspi"onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Server', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });" class="p-button">Download Ubuntu Server 23.10</a>
+        &nbsp;
+        <span class="u-text--muted">ISO image, 1.1GB</span>
       </p>
     </div>
   </div>
-</section>
-
-<section class="p-strip--light is-shallow">
-  <div class="row u-vertically-center">
-    <div class="col-8 col-medium-3">
-      <h2>Certified for the Raspberry Pi</h2>
-      <p><a href="/certified">Certified devices</a> are tested for reliability and performance, ensuring you have the best out-of-the-box Ubuntu experience. Ubuntu LTS releases are certified on select Raspberry Pi hardware.</p>
-    </div>
-    <div class="col-4 u-align--center u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/f76dd871-ubuntu-certified.png",
-        alt="",
-        width="150",
-        height="212",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<section class="p-strip--light u-no-padding--top">
-  <div class="row u-vertically-center">
-    <div class="col-4 u-align--center u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
-        alt="",
-        width="250",
-        height="250",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+  <hr class="p-rule is-fixed-width" />
+  <div class="row p-section">
+    <div class="col-6">
+      <h2>Ubuntu {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr></h2>
```

Comment:
Please remove the `abbr` tag from here, let's not use it in the heading (for consistency and keeping it light)